presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -153,3 +153,8 @@ Hive, this table can be described in Presto::
 This table can then be queried in Presto::
 
     SELECT * FROM hive.web.page_view;
+
+Hive Connector Limitations
+---------------------------
+
+* :doc:`/sql/delete` : Only supports ``DELETE`` if the WHERE clause matches entire partitions.
```

Comment:
This formatting is a bit weird. Since we only have one item now, it should just be a normal sentence. We can figure out a different way to structure it later when there are multiple items.